lib/src/Uncertainty/Algorithm/Sensitivity/HSICEstimatorConditionalSensitivity.cxx

```diff
@@ -33,12 +34,11 @@ HSICEstimatorConditionalSensitivity::HSICEstimatorConditionalSensitivity()
 /* Constructor */
 HSICEstimatorConditionalSensitivity::HSICEstimatorConditionalSensitivity(
   const CovarianceModelCollection & covarianceList
-  , const Sample & X, const Sample & Y
-  , const HSICStat & estimatorType
+  , const Sample & X
+  , const Sample & Y
   , const Function & weightFunction)
-  : HSICEstimatorImplementation(covarianceList, X, Y, estimatorType)
+  : HSICEstimatorImplementation(covarianceList, X, Y, HSICVStat())
 {
-  if (!estimatorType_.isCompatibleWithConditionalAnalysis()) throw InvalidArgumentException(HERE) << "You must use an appropriate HSICStat object for ConditionalSensitivity";
```

Comment:
seems isCompatibleWithConditionalAnalysis method is useless now

Reply:
indeed

Reply:
But I will not drop it